src/openapi/streaming/streaming.ts

```diff
@@ -184,6 +186,7 @@ class Streaming extends MicroEmitter<EmittedEvents> {
         this.baseUrl = baseUrl;
         this.authProvider = authProvider;
         this.transport = transport;
+        this.onProbe = options?.onProbe;

```

Comment:
I expected this to do

this.trigger('probe')

rather than expect a new option for a callback.

Reply:
Agreed. Have made that change and tested it locally.